packages/wheelhouse-frontend/src/components/environment-variables/EnvironmentVariablesDisplay.css

```diff
@@ -0,0 +1,38 @@
+
+.environment-variable-container {
+  margin-top: 30px;
+  margin-left: 0px;
+}
+.buttonText {
+  color: black;
+}
+.variable-container {
+  border: 1px black;
+}
+.variable-name {
+  color: black;
+  text-align: left;
+  border: 1px;
+
+}
+.envContainer {
+  margin-bottom: .4em;
+}
+.customInput {
+  width: 30em !important;
```

Comment:
note to self: figure out why everyone has to be combating Milligram here and fix it